asteroid/data/__init__.py

```diff
@@ -3,4 +3,4 @@
 from .dns_dataset import DNSDataset
 from .librimix_dataset import LibriMix
 from .wsj0_mix import Wsj0mixDataset
-from .avspeech_dataset import AVSpeechDataset
+from .avspeech_dataset import AVSpeechDataset, get_frames
```

Comment:
We won't be able to have this import because of all the dependencies in avspeech_dataset.py. 
Those will be extra_requirements and installed in the recipe.

Reply:
I see. Where should I put this file? Inside `egs/avspeech/`?

Reply:
No no, it can stay here. Just you'll have to import it as
`from asteroid.data.avspeech_dataset import ...`